src/middlewares/openapi.request.validator.ts

```
@@ -218,7 +227,12 @@ export class RequestValidator {
           message: message,
         });
         error.errors = err.errors;
-        throw error;
+        if (this.requestOpts.onError) {
+          this.requestOpts.onError(error, req);
+          next();
```

do we need to call next() here? we should expect onError handlers to always throw, correct? is the intent to allow clients to eat errors?